content/quick_start.md

```diff
@@ -43,18 +43,11 @@ For the quickest way to get started using Chef Infra:
     chef-client --local-mode --override-runlist first_cookbook
     ```
 
-This will create a file named `test.txt` at the home path on your
-machine. Open that file and it will say
-`This file was created by Chef Infra!`.
+This will create a file named `test.txt` at the home path on your machine. Open that file and it will say `This file was created by Chef Infra!`.
```

Comment:
```suggestion
This creates a file named `test.txt` at the home path on your computer. Open that file and it will say `This file was created by Chef Infra!`.
```